src/apps/ScopeChangeRequest/Components/Workflow/Criteria/Components/Criteria.tsx

```diff
@@ -61,10 +63,27 @@ export const CriteriaRender = ({
             </WorklowIconAndLine>
             <WorkflowRow>
                 <RowContent>
-                    {state && state.criteriaId === criteria.id ? (
+                    {state && state.criteriaId === criteria.id && state.action === 'Reassign' ? (
                         <CriteriaActionOverlay />
                     ) : (
                         <>
+                            {state &&
+                                state.criteriaId === criteria.id &&
+                                state.action !== 'Reassign' && (
```

Comment:
Lettere å lese hvis man drar det ut i en egen funksjon utenfor komponenten
```suggestion
                            {isNotReassign(state) &&  (
```